mopidy/core/library.py

```diff
@@ -271,20 +272,21 @@ def search(self, query=None, uris=None, exact=False, **kwargs):
             # "file:///media/music" and "spotify:"
             search({'any': ['a']}, uris=['file:///media/music', 'spotify:'])
 
-            # Returns results matching artist 'xyz' and 'abc' in any backend
-            search({'artist': ['xyz', 'abc']})
+            # Sample text based search
+            search('a artist:xyz "foo bar"')
 
         :param query: one or more queries to search for
-        :type query: dict
+        :type query: dict or a string
         :param uris: zero or more URI roots to limit the search to
-        :type uris: list of string or :class:`None`
-        :param exact: if the search should use exact matching
-        :type exact: :class:`bool`
+        :type uris: list of strings or :class:`None`
         :rtype: list of :class:`mopidy.models.SearchResult`
 
         .. versionadded:: 1.0
             The ``exact`` keyword argument, which replaces :meth:`find_exact`.
 
+        .. versionchanged:: 1.1
```

Comment:
> versionchanged:: version
Similar to versionadded, but describes when and what changed in the named feature in some way (new parameters, changed side effects, etc.).

The "new parameters" is the reason this confuses me.